larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -586,21 +729,48 @@ INSTANTIATE_TEST_SUITE_P(
         ::testing::Values(Padding_VALID, Padding_SAME, Padding_ONE),  // padding
         ::testing::Values(ActivationFunctionType_NONE,
                           ActivationFunctionType_RELU),  // activation function
-        ::testing::Values(1, 2),                         // number of threads
+#if RUY_PLATFORM(ARM_64)
+        // There's not yet Arm64 kernel support for bitpacked activations.
+        ::testing::Values(false),  // read bitpacked input
+        ::testing::Values(false),  // write bitpcked output
+#else
+        ::testing::Values(false, true),  // read bitpacked input
+        ::testing::Values(false, true),  // write bitpacked output
+#endif
```

Comment:
Just a reminder, when the arm-64 case falls back to the standard cpp kernel as per another comment, we can remove this `#if` block here.